client.go

```diff
@@ -285,11 +277,15 @@ func (cli *Client) LogRequestDone(req *http.Request, resp *http.Response, handle
 	path = strings.TrimPrefix(path, "/_matrix/client")
 	evt := zerolog.Ctx(req.Context()).Debug().
 		Str("method", req.Method).
+		Str("url", req.URL.String()).
```

Comment:
I'm not sure if we want to include this or not. It was included in the log at the start of the request, so I thought it might be good to include here as well.

@tulir any thoughts?

Reply:
Yeah should probably be here, makes `path` redundant though